### ISSUE TITLE
Not working on v17.09 build 16299.19

### ISSUE THREAD

**Issue by kareltdlg** [onlineth/Spotlight-Desktop#7]
When restart my pc after install the 1.3.3 version, Windows show me an error:

 spotlightdesktop | error
Found the registry information but it did not include a path to the Spotlight image location.

Please, I really like this feature on my pc.

**Comment by onlineth** [comment]
Hmm, that's interesting, I know of one case where this happened on my buddies system, you have the Spotlight wallpaper chosen for your lock screen on **your** user account, right? He didn't and when he turned it on, it worked. When you log in and lock your screen, you should see not only the spotlight image but also those small text boxes around the image.


If you did that and know for sure that your lock screen is using the spotlight image, could you let me know, are you using a 64bit version of Windows and do you know how to basically navigate regedit?

**Comment by kareltdlg** [comment]
Yes, I have chosen Spotlight for my lock screen and can see the text boxes around the images. Also, I am using a local account instead of a microsoft account, I don't know if that is a problem .
I am using a 64bit version and I know to navigate regedit.

**Comment by onlineth** [comment]
Ok, so essentially, we need to find the location where you see something to the effect of this.

![image](https://user-images.githubusercontent.com/7810121/32137071-b95c7e24-bbde-11e7-830c-73fdb9b2a4cb.png)

So far, I know of three locations where it could be. The script checks three locations (apparently there are four if we include yours:/).

1.  HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Authentication\LogonUI\Creative\\[An ID]\\[Sub node (whichever is the last one)]\
2. HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Authentication\LogonUI\Creative\\[An ID]  - (No subnodes)
3. And the old registry location that's being phased out SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Lock Screen\\Creative

Can you search around those locations, and see if you find a something that includes something to the effect of Landscape Asset path and has the value that includes a location to your appdata directory?

If you do, can you include a screenshot that includes the whole registry window and paste that path to that registry location?

**Comment by kareltdlg** [comment]
Sorry, I can find my path. LandscapeAssetPath values in regedit is empty.
![path](https://user-images.githubusercontent.com/30301939/32137630-0ac70052-bbfa-11e7-896e-d0c0b84de822.png)

**Comment by onlineth** [comment]
Oh, I see, actually, that might be ok. If you go into one of the paths in that subtree, what do you see there?

![image](https://user-images.githubusercontent.com/7810121/32137652-89c9475e-bbe9-11e7-878f-7043ccd4b08a.png)

**Comment by kareltdlg** [comment]
I see the same features in all three folder. I checked this is my current lock screen image.  

![path1](https://user-images.githubusercontent.com/30301939/32137726-e919836a-bbfb-11e7-9c25-a426dc0341c9.png)

**Comment by onlineth** [comment]
Ok, good news, I'll have to move some parts that check for the locations around in the script but that's the correct location.

Unfortnelty, I have some heavy midterms coming around this week and one next week so I really should focus on those but if I have some spare time, I'll update it accordingly, if not, by November 8th I'll have it corrected.

**Comment by kareltdlg** [comment]
Thanks for your support.

**Comment by onlineth** [comment]
@kareltdlg Almost forgot to add, as a temporary fix, copy the path found in `landscapeImage` and paste that value into the name `LandscapeAssetPath` (which is currently empty). Then reboot and you should be good to go for now.

**Comment by kareltdlg** [comment]
Great!! That works for me. It's a good temporary solution. Thanks.

**Comment by onlineth** [comment]
Yep, it just checks that location because multiple users will have multiple of those nodes and the image path is relative to that node. So one node will contain image paths to my personal app data folder and another node will have image paths to different user's app data folder.

![image](https://user-images.githubusercontent.com/7810121/32147163-9a029900-bcb0-11e7-9b2c-bab7877bdf7f.png)

See how I have two of those long named nodes under creative on the left? One of them is for a separate user account on my system.

I'll still work on getting a fix though moving forward.

**Comment by MikeStrider** [comment]
This resolved my issue aswell. Manually created the following KEY.
Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Authentication\LogonUI\Creative\S-1-5-21-..........

add STRING like this = C:\Users\mikes\AppData\Local\Packages\Microsoft.Windows.ContentDeliveryManager_cw5n1h2txyewy\LocalState\Assets\b2d9e2dc199b....d39e630f61f847b6fd2.......6fa6ffbe805d8e9d3e

Thank you **onlineth** for the great app and support !!!!

**Comment by onlineth** [comment]
@MikeStrider Great to hear, I will fix this in the future once I catch a break from my studies.

**Comment by myworld122** [comment]
Thanks guys, this helps!

**Comment by onlineth** [comment]
@myworld122 A pre-release is out which first checks those second nested subkeys without looking initially at the LandscapeAssetPath. Let me know if this fixes your issue,

https://github.com/onlineth/Spotlight-Desktop/releases/tag/2.0.0
